_rules/links-with-identical-names-and-context-serve-equivalent-purpose-fd3a94.md

```diff
@@ -229,80 +234,124 @@ These two HTML `a` elements have the same [accessible name][] and [context][prog
 </html>
 ```
 
+### Failed
+
+#### Failed Example 1
+
+These two HTML `a` elements have the same [accessible name][] and [context][programmatically determined link context]. They are visually distinguishable thanks to the relationships conveyed through CSS, but go to different resources.
+
+```html
+<html lang="en">
+	<p>
+		<h2>Contact us:</h2>
+		<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/contact-us.html?page=1" style="display:inline-block; background: url(/test-assets/shared/chat.png) 0 / 40px no-repeat; padding: 20px 0 20px 50px;">Contact Us</a>
+		<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/contact-us.html?page=2" style="display:inline-block; background: url(/test-assets/shared/phone.png) 0 / 40px no-repeat; padding: 20px 0 20px 50px; margin-left: 40px;">Contact Us</a>
+	</p>
+</html>
+```
```

Comment:
I feel this also fails 1.3.1 by not conveying programmatically the info conveyed by the background images. Likely also fails1.1.1 since these are informative images without text alternative…

That's not really a problem, but then I guess we should have 1.1.1 and 1.3.1 as secondary requirements with a Background note explaining why.

Reply:
Added the following background note:

> This rule contains Failed Examples that use background informative images without an alternative text to convey information. This leads also [success criterion 1.1.1 Non-text Content][sc111] to fail. Similarly, the visual position of the elements allows sighted users to disambiguate the target links thanks to their vertical alignment with the images above. This causes issues under [success criterion 1.3.1 Info and Relationships][sc131] as relationships conveyed visually are not programmatically determined.

Reply:
Should I add also 1.1.1 and 1.3.1 to the _Bibliography_ section?

Reply:
Indeed, looking at the other rules it seems we are putting the secondary requirements in the Bibliography.

Reply:
I've added them into the Bibliography section adding "(secondary requirement)" to clarify why they are there. Maybe we can find a more meaningful way, but I just put it there for now.